docs/data/sql_functions.yml

```diff
@@ -350,6 +350,17 @@ string:
   - sql: LOCATE(string1, string2[, integer])
     table: STRING1.locate(STRING2[, INTEGER])
     description: Returns the position of the first occurrence of string1 in string2 after position integer. Returns 0 if not found. Returns NULL if any of arguments is NULL.
+  - sql: URL_DECODE(string)
```

Comment:
One thing I forgot to mention: we also need to update Chinese version (it could be in English however should be synced)

Reply:
ok, I will add